src/main/scala/beam/experiment/ExperimentRunSandbox.scala

```diff
@@ -13,7 +13,6 @@ case class ExperimentRunSandbox(projectRoot: Path, experimentBaseDir: Path, expe
   lazy val runConfig: Config = buildRunConfig
 
   def runDirectory = Paths.get(experimentBaseDir.toString,
-    experimentDef.title.replace("\\s", "_"),
```

Comment:
I'd better leave replace in case user want to put whitespaces in title, otherwise it may cause file not found error while resolving runExperiment.sh, etc.